lib/client/ws_client.rb

```diff
@@ -21,9 +20,7 @@ def self.start(url)
               data = message.parse
               ScaleRb.logger.debug "Received message: #{data}"
 
-              task.async do
-                client.handle_response(data)
-              end
+              client.handle_response(data)
```

Comment:
@ioquatix I have some qustions about how to handle error here. 

1. Handle all StandardError errors raised from `client.handle_response(data)`?
2. `connection.read` and `message.parse` would raise StandardError. Catch and log these errors?

Reply:
For (1), I suppose it depends on whether it invokes user code and whether it's a fatal error.

For (2), yes the connection may fail due to `IOError` or some other problem (e.g. remote server restarts, etc). You need to decide if you want to reconnect, or something else, e.g. log an exit.